armi/utils/tests/test_utils.py

```diff
@@ -17,6 +17,7 @@
 """
 import unittest
 import math
+import os
 import numpy as np
```

Comment:
should leave the single blank line between python built in imports and third party ones.